assets/readme.md

```diff
@@ -1,19 +1,121 @@
-This ship enriches customer profiles using [Clearbit](https://clearbit.com)
+# Hull Clearbit Connector
 
-Fetch 3rd party data from Clearbit and writes it back into the customer profile.
+The Clearbit Connector enables sales and marketing teams to enrich customer profiles, reveal anonymous website traffic, find the ideal buyer and discover new accounts.
 
-# Setup
+## Getting Started
 
-- Go to your [Clearbit dashboard, under API keys](https://dashboard.clearbit.com/keys). Copy the key, Paste it into your Hull Dashboard.
-- Go to your [Clearbit Account Settings](https://dashboard.clearbit.com/account). Under Webhook URL, enter https://hull-clearbit.herokuapp.com/clearbit
+Go to the Connectors page of your Hull organization, click the button “Add Connector” and click “Install” on the Clearbit card. After installation, switch to the “Settings” tab and begin with the configuration.
 
+Begin your configuration in the section **Authentication** by copying the Clearbit API key into the provided field. If you are not sure which key to use, see [Find Your Clearbit API Key](#Find-Your-Clearbit-API-Key) for details.
+![Getting Started Step 1](./docs/gettingstarted01.png)
+Your Clearbit Connector is now ready to be configured depending on your needs, see Features to learn how to configure the various solutions.
 
-## Similar Lead discovery
+## Features
 
-The lead discovery feature uses [Clearbit Discovery](https://clearbit.com/discovery) and [Clearbit Prospector](https://clearbit.com/prospector) to find new leads based your existing users.
+Clearbit [**Enrichment**](#Enrichment) enables your team to transform the email address of an user in Hull into a complete user and company profile with up to 85 data points.
```

Comment:
"transform" stays, it is not good grammar to repeat enrich two times within the first half of a sentence and this is also how Clearbit describes Enrichment. We should not reinvent the description of features from other products... I fixed the "an" spelling mistake to "a".